src/main/java/nikki/task/Task.java

```diff
@@ -0,0 +1,204 @@
+package nikki.task;
+
+import nikki.NikkiException;
+
+public class Task {
+    /** Mark in checkbox of completed Task */
+    private static final char completedMark = 'X';
```

Comment:
Could use uppercase variable names to store constant

Reply:
Thanks for pointing this out!